source/content/environment-specific-config.md

```diff
@@ -38,20 +37,21 @@ if (!defined('WP_DEBUG') && isset($_ENV['PANTHEON_ENVIRONMENT'])) {
 
 <Alert title="Warning" type="danger">
 
-PHP constants like `WP_DEBUG` can only be defined once. When implementing this code snippet, remove or comment out the [existing code block](https://github.com/pantheon-systems/WordPress/blob/master/wp-config.php#L147) defining it.
+PHP constants like `WP_DEBUG` can only be defined once. When implementing this code snippet, remove or comment out the [existing code block](https://github.com/pantheon-systems/WordPress/blob/default/wp-config.php#L74) defining it.
```

Comment:
The comment in the file says that you should just change the `false` to `true` right there. Should our documentation say the same thing?

Reply:
We could perhaps suggest changing it to:
```
if ( ! defined( 'WP_DEBUG' ) ) {
	define('WP_DEBUG', (wp_get_environment_type() == "development"));
}
```
Just setting it to `true` isn't great, because then you have to remember to set it back to `false` before deploying every time. This way debug mode would always be enabled in dev / multidev, but not in test / live.

Reply:
I updated this with your suggestion, and I'm going to leave it open (not mark it Resolved) so that whoever is copyediting has this for reference.